### PR TITLE
Linea soak

### PR DESCRIPTION
**Pull request by davidcauchi**

### CODE REVIEW

**Review by simsonraj — APPROVED** [review]

**Review by jmank88 — DISMISSED** [review]

File: core/chains/evm/config/toml/defaults/Linea_Mainnet.toml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+[Transactions]
+# increase resend time to align with finality
+ResendAfterThreshold = '3m'
+
+# set greater than finality depth
+[HeadTracker]
+HistoryDepth = 350
+
+# Linea uses 200 blocks to estimate gas price
```
Comment (prashantkumar1982) [inline core/chains/evm/config/toml/defaults/Linea_Mainnet.toml:19]: 200 seems very large.
Is this depth recommended by Linea?

**Review by prashantkumar1982 — COMMENTED** [review]
Looks good, just a question on the blockhistoryestimator size

**Review by prashantkumar1982 — APPROVED** [review]

File: core/chains/evm/config/toml/defaults/Linea_Mainnet.toml (hunk 1 above)
Comment (dimriou) [reply core/chains/evm/config/toml/defaults/Linea_Mainnet.toml:19]: Indeed. I see ~15 txs per block on Mainnet, which translates to 3000 txs for the estimator. In comparison, ETH Mainnet has ~150txs per block and the fallback value is 8.

Comment (davidcauchi) [reply core/chains/evm/config/toml/defaults/Linea_Mainnet.toml:19]: At analysis stage the Linea docs mentioned that gas was calculated on the last 200 blocks. For this reason `BlockHistorySize` was set to 200. Should be noted that this detail has now been removed from the docs.

**Review by jmank88 — APPROVED** [review]

**Review by simsonraj — APPROVED** [review]

**Pull request merged**